hooks/environment.agent

```diff
@@ -9,7 +9,7 @@ set -eou pipefail
 ## agent should exit, causing the docker container to restart and restore the deleted files.
 ## This gives us the capability to deny access to these files to later steps within the
 ## current buildkite job.
-SECRETS_MOUNT_POINT="${BUILDKITE_PLUIGIN_CRYPYTIC_SECRETS_MOUNT_POINT:-/secrets}"
+SECRETS_MOUNT_POINT="${BUILDKITE_PLUGIN_CRYPTIC_SECRETS_MOUNT_POINT:-/secrets}"
```

Comment:
I was wondering how a typo survived `set -u`, but the `${...:-...}` syntax to provide a fallback of course sidesteps `-u`, sigh.